components/Layout/Navbar.tsx

```diff
@@ -143,6 +143,7 @@ const Navbar: FC = () => {
       >
         <Link href={RoutePaths.Home} passHref>
```

Comment:
I think this Link should be moved in to wrap the Logo instead of the Box. As is with this change, it causes the `NodeVersionButton` to also link to the homepage when right-clicking on it to open in a new tab,

Reply:
Good catch. I did this and also tweaked some flex stuff so the Iron Fish link would be bigger and easier to click.

Reply:
Oh nice, I like that better!